docs/usecases/remodnav.rst

```diff
@@ -291,45 +290,137 @@ to fill tables with unique names with minimal work, for example like this:
      \end{tabular*}
    \end{table}
```

Comment:
It would probably be sufficient to trim this down to one result column and (maybe) two sections for better readabiity and formatting in the PDF version. The table screenshot could be cropped to match.